test/conftest.py

```diff
@@ -713,12 +706,12 @@ def ensure_ids_are_correct_uuids(docs: list, document_store: object) -> None:
 
 
 @pytest.fixture(params=["elasticsearch", "faiss", "memory", "milvus1", "milvus", "weaviate", "pinecone"])
-def document_store_with_docs(request, test_docs_xs, tmp_path):
+def document_store_with_docs(request, tmp_path):
```

Comment:
Using `DOCS` instead of `test_docs_xs` here is no problem but keep an eye out for tests that make changes to the documents, for example, adding metadata. Reusing the same DOCS in tests after they were change could lead to problems.

Reply:
Right! I'll double check. It can also be that pytest isolates the tests, so such changes can't carry over, but I have no idea if that's actually the case. I'll test.

Reply:
This passes, so I believe there is no reason for concern :slightly_smiling_face: 

```python
from haystack.schema import Document

DOCS = [Document("hello")] * 10

def test_1():
    DOCS[1].content = "hi"
    assert True

def test_2():
    assert [doc.content == "hello" for doc in DOCS]
```

Reply:
You're right, this fails:
```python
from haystack.schema import Document

DOCS = [Document("hello")] * 10

def test_1():
    DOCS[1].content = "hi"
    assert True

def test_2():
    for index, doc in enumerate(DOCS):
        print(index)
        assert doc.content == "hello"
```
I'll revert unless I come up with something else.

Reply:
Reverted to fixtures, indeed it was the cleanest choice from the start. Thanks for spotting this!